### PR TITLE
Desktop version of personal portfolio

### PR DESCRIPTION
**Pull request by mauriciomonta**
In this case I already worked on desktop version before this week, For this rason I just add some lines to create pull request.
The desktop version it's working and I think I reached all the objectives of this excercise.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -158,6 +158,99 @@ a:hover {
...
+
+#portfoliosection {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+#aboutmesection {
```
Comment (Deepakdanger) [inline style.css:183]: - [ ] The background image of the `about me` section in `desktop mode` is not aligned properly, Kindly fix it.
- Original website
bfvbc
- Your website
fv1

File: style.css [hunk 2]
```diff
@@ -307,6 +400,85 @@ header {
   display: none;
 }
 
+#contact {
```
Comment (Deepakdanger) [inline style.css:403]: - [ ] The background image of the `footer` section in desktop mode is not aligned properly, Kindly fix it. Check the mobile version section also, the background in the mobile view `section should not get affected by the change.
- Original website
ffedf4
- Your website
vsd2

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @mauriciomonta, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (mauriciomonta) [reply style.css:183]: I'm seeing the images where it should be. I don't know what happen 😢.

Captura de pantalla 2022-01-15 181830

File: style.css (hunk 2 above)
Comment (mauriciomonta) [reply style.css:403]: Same thing in this case 😭. Here I should do some position changes but I see it so different than you.

Captura de pantalla 2022-01-15 181844

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM👍, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉🎉
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**